cmake/externals/projects_modules/VTK.cmake

```diff
@@ -51,7 +51,7 @@ EP_SetDirectories(${ep}
 # Set GIT_TAG to latest commit of origin/release-5.10 known to work
 set(tag tags/v5.10.1)
 if (NOT DEFINED ${ep}_SOURCE_DIR)
-    set(location GIT_REPOSITORY "git@github.com:Kitware/VTK.git" GIT_TAG ${tag})
+    set(location GIT_REPOSITORY "https://github.com/Kitware/VTK.git" GIT_TAG ${tag})
```

Comment:
@LoicCadour you changed this line in https://github.com/Inria-Asclepios/medinria-superproject/pull/12
Can you test or look at this new version?